assemblies/tuning-a-model-by-using-lora.adoc

```diff
@@ -0,0 +1,25 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+
+[id="tuning-a-model-by-using-lora_{context}"]
+= Tuning a model by using LoRA
+
+[role='_abstract']
+In {productname}, you can use LoRa (Low-Rank Adaptation) to efficiently fine-tune large language models, such as Llama 3. 
```

Comment:
Did you mean to use  {productname-short} ?

Reply:
I did indeed, thank you! Fixed now :)